pkg/config/config.go

```diff
@@ -366,7 +366,13 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("dogstatsd_stats_port", 5000)
 	config.BindEnvAndSetDefault("dogstatsd_stats_enable", false)
 	config.BindEnvAndSetDefault("dogstatsd_stats_buffer", 10)
+	// Control for how long counter would be sampled to 0 if not received
```

Comment:
nit: `sampled` doesn't have a clear and consistent meaning IMO, and the way we use it in the agent means "a point is seen", could you change this to:

```suggestion
	// Control for how long dogstatsd counters are sent with `0` values if not received
```